src/gui/shell.cpp

```diff
@@ -725,9 +725,13 @@ void Shell::handleNeovimNotification(const QByteArray &name, const QVariantList&
 			m_mouseHide = variant_not_zero(args.at(1));
 			int val = m_mouseHide ? 1 : 0;
 			m_nvim->api0()->vim_set_var("GuiMousehide", val);
-		} else if (guiEvName == "Close" && args.size() == 1) {
+		} else if (guiEvName == "Close" && args.size() == 2) {
```

Comment:
Allow for size 1 or 2 to remain compatible with older versions i.e. return 0 if the argument is not present.